### ISSUE TITLE
Unnecesssary scenarios

### ISSUE THREAD

**Issue by aysnrarsln** [mjposs/min-max-min#2]
https://github.com/mjposs/min-max-min/blob/e89b1dd3e796308afc0a86f709b576ba95cddb87/Applications/KP.jl#L252

These k-1 scenarios added to cc and aa are not necessary. As is, it just repeats the nominal scenario k times. Unless we require k initial scenarios somewhere later in the code, we may get rid of this. I think it clutters the code...

**Comment by mjposs** [comment]
Yes I know and I don't remember why we needed it at some point. Maybe not needed anymore? I let you verify?
